packages/turbo/src/stradaBridge.ts

```diff
@@ -86,7 +84,7 @@ const stradaBridgeScript = `
       ${
         Platform.OS === 'android'
           ? 'StradaNative.postMessage(JSON.stringify(message))'
-          : 'webkit.messageHandlers.stradaNative.postMessage(message)'
+          : 'webkit.messageHandlers.nativeApp.postMessage(message)'
```

Comment:
Changing `userContentController` name to remove unnecessary `stradaNative` `userContentController` on native side.